lib/game.rb

```diff
@@ -1,24 +1,18 @@
 class Game
     attr_reader :game_id, 
                 :season, 
-                :type,
```

Comment:
All this YAGNI was making my code smell go off. Glad we got to review this final refactor as a group.